tensorflow/lite/micro/micro_mutable_op_resolver.h

```diff
@@ -443,6 +443,29 @@ class MicroMutableOpResolver : public MicroOpResolver {
                       ParseResizeNearestNeighbor);
   }
 
+  TfLiteStatus AddRfft() {
+    // TODO(b/286250473): change back name and remove namespace
+    return AddCustom("SignalRfft", tflite::tflm_signal::Register_RFFT());
+  }
+
+  TfLiteStatus AddRfftFloat() {
+    // TODO(b/286250473): change back name and remove namespace
+    return AddCustom("SignalRfftFloat",
+                     tflite::tflm_signal::Register_RFFT_FLOAT());
+  }
+
+  TfLiteStatus AddRfftInt16() {
+    // TODO(b/286250473): change back name and remove namespace
+    return AddCustom("SignalRfftInt16",
+                     tflite::tflm_signal::Register_RFFT_INT16());
+  }
+
+  TfLiteStatus AddRfftInt32() {
+    // TODO(b/286250473): change back name and remove namespace
+    return AddCustom("SignalRfftInt32",
+                     tflite::tflm_signal::Register_RFFT_INT32());
+  }
```

Comment:
Instead of separate functions, let's have the TFLMRegistration be a parameter similar to https://github.com/tensorflow/tflite-micro/blob/1355b89368ff1168f77af1bde9b4569ef2e5b353/tensorflow/lite/micro/micro_mutable_op_resolver.h#L264-L268

And then the callee would do something like:
```cc
AddRfft(Register_RFFT_INT32());
```

Reply:
Done. This looks much better.